app/javascript/menu/first-level.jsx

```diff
@@ -58,7 +59,7 @@ MenuItem.defaultProps = {
 };
 
 const MenuSection = forwardRef(({
-  active, expanded, hover, icon, id, items, title, onSelect,
+  active, expanded, hover, icon, id, items, title, onSelect, idx,
```

Comment:
Minor, but I'm not a fan of abbreviation at the API level.  Can we spell this out to `index`, or perhaps use another work like `position`?